package.json

```diff
@@ -4,6 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "build": "eleventy",
```

Comment:
Easier for me to build the site if I can just do <kbd>npm run build</kbd> from the root directory here (plus, it shows up as an "npm script" in my VS Code).